providers/azureblob/src/main/java/org/jclouds/azureblob/AzureBlobClient.java

```diff
@@ -254,4 +257,8 @@
     */
    boolean blobExists(String container, String name);
 
+   /**
+    * @throws ContainerNotFoundException if the container is not present.
+    */
+   void copyBlob(URI copySource, String toContainer, String toName, Optional<CopyBlobOptions> options);
```

Comment:
Changed to use a single non-`Optional` parameter and added `CopyBlobOptions.NONE`.